appveyor.yml

```
@@ -38,7 +45,19 @@ deploy:
 
   - provider: GitHub
     tag: $(APPVEYOR_REPO_TAG_NAME)
+    description: ''
     auth_token:
         secure: Otbl8p8qCwciDqJgSWCyN0Arfs5XS1CwiHcK+r0F6uz9Rxt4gzBFvlc3cjPV3NxR
     on:
-        APPVEYOR_REPO_TAG: true
\ No newline at end of file
+        APPVEYOR_REPO_TAG: true
+
+after_deploy:
+  - ps: |
+      # Only run this script on tagged builds
+      if ($env:APPVEYOR_REPO_TAG -ne "true") { return }
+
+      # Push NestingContently Umbraco package
+      $path = $artifacts["UmbPack_NestingContently"].path
+      if ($path) {
+        umbpack push $path -k $env:UmbPackKey -a current
+      }
```

To push both packages to Our Umbraco, this can be changed to:
```suggestion
      # Push NestingContently Umbraco package and archive all existing packages
      $path = $artifacts["UmbPack_NestingContently"].path
      if ($path) {
        umbpack push $path -k $env:UmbPackKey -a *
        
        # Also push NestingContently ValueConverters Umbraco package without setting it as the new current version
        $path = $artifacts["UmbPack_NestingContently_ValueConverters"].path
        if ($path) {
          umbpack push $path -c false -k $env:UmbPackKey
        }
      }
```

---

Just note that I haven't fully tested this, so it might need some tweaking 😇

---

I've updated the suggesion to only archive the previous ValueConverters packages, so versions 1 and 2 of the base package will not be archived 😉